index.d.ts

```diff
@@ -0,0 +1,197 @@
+import {Options as MinimistOptions} from 'minimist-options';
+
+export interface Options {
+	/**
+	Define argument flags.
+
+	The key is the flag name and the value is an object with any of:
+
+	- `type`: Type of value. (Possible values: `string` `boolean`)
+	- `alias`: Usually used to define a short flag alias.
+	- `default`: Default value when the flag is not specified.
+
+	@example
+	```
+flags: {
+	unicorn: {
+		type: 'string',
+		alias: 'u',
+		default: 'rainbow'
+	}
+}
```

Comment:
Sorry for being unclear, this should be indented the same as the backticks.

```
	```
	flags: {
		unicorn: {
			type: 'string',
			alias: 'u',
			default: 'rainbow'
		}
	}
```